-------------------------------------------------------------------------------
Test set: myTestRunner.TestRunner
-------------------------------------------------------------------------------
Tests run: 2, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 73.74 s <<< FAILURE! -- in myTestRunner.TestRunner
myTestRunner.TestRunner.runScenario["User is able to delete the product", "Purchase the order from Ecommerce website"](2) -- Time elapsed: 35.51 s <<< FAILURE!
java.lang.AssertionError: expected [false] but found [true]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertFalse(Assert.java:79)
	at org.testng.Assert.assertFalse(Assert.java:89)
	at stepDefinitions.SubmitOrderStepDef.verify_confirmationmsg(SubmitOrderStepDef.java:105)
	at âœ½.verify the confirmation message order deleted succefully is diplayed(file:///C:/Users/hemaj/eclipse-workspace/CuCumberBDDMaveProject/src/test/java/bddcucumber/SubmitOrder.feature:21)

